user_login

```diff
@@ -0,0 +1,22 @@
+<?php
+  function OpenCon()
+  {
+  $dbhost = "localhost";
+  $dbuser = "root";
+  $dbpass = "";
+  $db     = "hu_social";
+  $conn = new mysqli ($dbhost, $dbuser, $dbpass, $db) or die("Connect failed:%s\n".
+  $conn -> error);
+  return $conn;
+}
+function CloseCon($conn)
+{$conn -> close();
+}
+
+?>
+<?php
+include 'db_connection.php';
```

Comment:
This file is missing

Reply:
Oops, I'll edit this when I get home since the 'db_connection.php' file is on my other computer.